data/core/commands/doc.lua

```diff
@@ -67,6 +67,7 @@ local function cut_or_copy(delete)
       doc().cursor_clipboard[idx] = ""
     end
   end
+  doc().cursor_clipboard["full"] = full_text
```

Comment:
I am wondering here why we use the syntax ["full"] instead of .full.

I suggest also using a different name as "full" doesn't seem very meaningful to me. What about calling it "save" or something like that. In any case the choice is up to you.

Reply:
I'm basically calling it 'full', because in this same hash, using a numeric index is a partial clipboard for a particular line. That's where it's coming from. "save" to me has no relation to the indexed clipboards, so I think is less relevant than "full".

I could call it "full_clipboard" or "entire_clipboard"  for clarity's sake, if you like; but honestly, I feel it's good as-is in the context of the code.